src/pages/Dashboard.tsx

```diff
@@ -0,0 +1,261 @@
+import React, { FC } from "react"
+import VideoGrid from "layouts/VideoGrid"
+
+const Dashboard: FC = () => {
+	return (
+		<>
+			<VideoGrid
```

Comment:
Layouts should not be used like normal components.
Please update the layouts to only define layout of pages that have a common layout.